lib/_http_server.js

```diff
@@ -485,8 +485,10 @@ function connectionListener(socket) {
       res.assignSocket(socket);
     }
 
+    socket.setTimeout(self.timeout);
```

Comment:
In this place we renew ```timeout``` after ```keepAliveTimeout``` when new requests came. So I think we need:
```JavaScript
if (self.timeout) socket.setTimeout(self.timeout);
```
By the way, core http libs have different code styles mixed, even ```_http_server.js``` in different places have mixed style. So do we have code templates, best practices or code style standards? For example, how should I do following:
```JavaScript
// I prefer such style for short lines
if (self.timeout) socket.setTimeout(self.timeout);

// But sometimes use this
if (self.timeout) {
  socket.setTimeout(self.timeout);
}

// And do not like such style, but can live with it
if (self.timeout)
  socket.setTimeout(self.timeout);
```